File: src/Neuralm.Services/Neuralm.Services.TrainingRoomService/Neuralm.Services.TrainingRoomService.Messages/PostOrganismsScoreResponse.cs
using Neuralm.Services.Common.Messages.Abstractions;
using System;

namespace Neuralm.Services.TrainingRoomService.Messages
{
    /// <summary>
    /// Represents the <see cref="PostOrganismsScoreResponse"/> class.
    /// </summary>
    public class PostOrganismsScoreResponse : Response
    {
        /// <summary>
        /// Initializes an instance of the <see cref="PostOrganismsScoreResponse"/> class.
        /// </summary>
        /// <param name="requestId">The request id.</param>
        /// <param name="message">The message.</param>
        /// <param name="success">The success flag.</param>
        public PostOrganismsScoreResponse(Guid requestId, string message = "", bool success = false) : base(requestId, message, success)
        {

        }
    }
}

File: src/Neuralm.Presentation.Web/src/messages/responses/PostOrganismsScoreResponse.ts
import Response from './Response';

/**
 * Represents the PostOrganismsScoreResponse class.
 */
export default class PostOrganismsScoreResponse extends Response {
  /**
   * Initializes a new instance of the PostOrganismsScoreResponse class.
   */
  constructor(id: string, requestId: string, dateTime: Date, message: string, success: boolean) {
    super(id, requestId, dateTime, message, success);
  }
}
